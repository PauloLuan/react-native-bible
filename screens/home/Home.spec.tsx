import React from 'react'
import {Button, Text, TextInput, View} from 'react-native'
import {render, waitFor} from '@testing-library/react-native'

import { Home } from '.'
// import theme from '../../styles/Theme'

describe('<Home />', () => {
  const testId = 'component-home-id'

  it('should render the Home component', () => {
    const {getByTestId, queryByTestId, toJSON} = render(<Home testId={testId} />)
    await waitFor(() => expect(queryByTestId(testId)).toBeTruthy())
    expect(toJSON()).toMatchSnapshot()
  })
})
