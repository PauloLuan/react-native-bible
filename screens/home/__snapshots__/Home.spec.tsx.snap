// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the Home component 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "blue",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "height": 60,
        "width": 85,
      },
    ]
  }
  testID="component-home-id"
>
  <Text>
    example
  </Text>
</View>
`;
